main.c

```diff
@@ -45,6 +53,9 @@ int main(int argc, char* argv[]) {
         printf("Failed to call extract (error %d)\n", result);
         exit(-1);
     }
+    //What does result return on failure and exit? Might be better to leave well enough alone.
+    //I'm lazy.
+
```

Comment:
Would either refactor `extract` to return a bool and exit(EXIT_FAILURE) or remove comment